projects/components/src/stepper/stepper.component.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  EventEmitter,
+  Input,
+  Output,
+  QueryList
+} from '@angular/core';
+import { MatStepper } from '@angular/material/stepper';
+import { queryListAndChanges$ } from '@hypertrace/common';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ButtonRole, ButtonStyle } from '../button/button';
+import { IconSize } from '../icon/icon-size';
+import { StepperTabComponent } from './stepper-tab.component';
+
+@Component({
+  selector: 'ht-stepper',
+  template: `
+    <div class="stepper" *ngIf="this.steps$ | async as steps">
+      <mat-stepper [orientation]="this.orientation" [linear]="this.isLinear" #stepper>
+        <ng-container *ngFor="let step of steps; index as i">
```

Comment:
Can we remove the index variable from here, it seems not necessary.

Reply:
missed that! thanks! 
updated.